docs/site/src/pages/reference/console/channels.mdx

```diff
@@ -0,0 +1,88 @@
+---
+layout: "@/layouts/MainLayout.astro"
+title: "Channels"
+heading: "Channels"
+description: "Store data in channels."
+---
+
+import { Divider } from "@synnaxlabs/pluto";
+import { Video } from "@/components/Media";
+import Table from "@/components/Table.astro";
+
+A [channel](/reference/concepts/channels) is a data store that represents ordered values
```

Comment:
I think the language `data store` is a bit confusing. We should stick to a consistent definition that is also defined in the concepts pages

Reply:
I changed it to use the same language as the concepts page. I don't love what it says "logical collection of time-ordered data", but I'm not sure what else to call it. I've described channels as the equivalent of columns in a traditional table database, but I don't think that's the best way to describe it, especially for Console users.